bin/tessel-2.js

```diff
@@ -277,6 +277,15 @@ parser.command('update')
   })
   .help('Update the Tessel firmware and openWRT image');
 
+parser.command('version')
+  .option('timeout', timeoutOption)
+  .option('name', nameOption)
```

Comment:
This command should also have the [`usb` and `lan` options](https://github.com/tessel/t2-cli/blob/master/bin/tessel-2.js#L74-L81). I wonder if there is a way to make the `timeout`, `name`, `lan`, and `usb` options common to all commands?